dapr/proto/runtime/v1/dapr.proto

```diff
@@ -654,4 +664,34 @@ message UnlockResponse {
   }
 
   Status status = 1;
-}
\ No newline at end of file
+}
+
+message WorkflowReference {
+  string InstanceID = 1;
+}
+
+message GetWorkflowRequest {
+  WorkflowReference WorkflowReference  = 1;
+  string WorkflowType = 2;
+  string InstanceID = 3;
```

Comment:
What's the difference between this `InstanceID` and `WorkflowReference.InstanceID`?